app/assets/stylesheets/components/_content-metadata.scss

```diff
@@ -0,0 +1,16 @@
+.app-c-content-metadata {
+  @include responsive-bottom-margin;
+  @include core-16;
+  padding-top: $gutter-half;
+  padding-bottom: $gutter-half;
+  border-top: 1px solid $border-colour;
+  border-bottom: 1px solid $border-colour;
+}
+
+.app-c-content-metadata__from {
+  margin-top: $gutter-half;
+}
+
+.app-c-content-metadata__from-link a {
```

Comment:
As this wraps multiple links, I'd use `.app-c-content-metadata__from-links`, but I think what was there before is still effectively the same as this, just less verbose.

Reply:
My plan was to apply that class to the links until I found it was a whole chain of magic to produce the links. Have reverted it now but un-embedded the anchor class. Hopefully can find time to work out passing through the class and improve this at some point.